core/citrus-api/src/main/java/org/citrusframework/report/TestListener.java

```diff
@@ -42,7 +42,7 @@ default void onTestFinish(TestCase test) {
      * @see #onTestEnd(TestCase)
      */
     default void onTestExecutionStart(TestCase test) {
-        onTestFinish(test);
+        // Default implementation does nothing
```

Comment:
😵

This was a last minute change using copy paste. The correct documentation and implementation should be:
(sorry for not being able to suggest on multiple lines - do not apply as patch)
```suggestion
    /**
     * Invoked when test execution starts (after {@link org.citrusframework.container.BeforeTest} execution)
     */
    default void onTestExecutionStart(TestCase test) {
        // Default implementation does nothing
    }
```

Reply:
thank you, I have fixed the javadocs, too. as you have suggested